src/screens/HtmlScreen.js

```diff
@@ -35,22 +38,13 @@ export const HtmlScreen = ({ navigation, route }) => {
     isConnected && title && trackScreenViewAsync(`${MATOMO_TRACKING.SCREEN_VIEW.HTML} / ${title}`);
   }, [title]);
 
-  if (!refreshTime) {
-    return <LoadingSpinner loading />;
-  }
-
-  const refresh = async (refetch) => {
+  const refresh = useCallback(async () => {
     setRefreshing(true);
-    isConnected && (await refetch());
+    isConnected && (await refetch?.());
     setRefreshing(false);
-  };
+  }, [refetch]);
```

Comment:
`}, [refetch, isConnected]);` as in `ServiceTiles`?

Reply:
you are correct. we do want the function to update when the connection state changes.